user_profile.php

```diff
@@ -0,0 +1,454 @@
+<?php
+require_once("utilities.php");
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Village X Org | Fund Projects That Villages Choose</title>
+<meta name="description" content="Disrupting extreme poverty in rural Africa with democracy, direct giving, and data."/>
+<?php include('header.inc'); ?>
+
+<div id="index-banner" class="parallax-container hide-on-med-and-down" style="background-color: rgba(0, 0, 0, 0.3); height: 500px">
+    
+	<div class="container no-pad-bot" style="height: 100%; width: 100%; padding: 2% 0% 0% 0%">
+		<div class="row">
+		  <div class="col s6 m6 l2 right-align" style="padding: 9% 0% 0% 3%;">
+			<div style="width:175px; height:175px; border-radius:20%; border-style:solid; background-color:#008080BB;">
+        				<h1 class="header center-align light text-shadow: 2px 2px 7px #111111" style="padding:12% 5% 5% 5%"><b>B</b></h1>
```

Comment:
You put a lot of style stuff in the class tag -- can you move these to the style tags, or remove them?